README.md

```diff
@@ -37,6 +37,16 @@ ape test --network ethereum:local:boa
 
 Boa works the same as other Ape testing providers such as `ape-foundry` or the Ethereum tester that comes with `ape-test` by default.
 
+### Fast Mode
+
+By default, `ape-titanoboa` runs boa in "fast mode".
+To disable fast mode, configure the plugin with `fast_mode: False`:
+
+```yaml
+titanoboa:
+  fast_mode: False
```

Comment:
Should the default be `True` or `False`? I guess in regular boa, the default is `False` so maybe should go with that...